server.js

```diff
@@ -37,10 +37,8 @@ let connStr = "DATABASE="+process.env.DB_DATABASE+";HOSTNAME="+process.env.DB_HO
 //let connStr = "DATABASE=BLUDB;HOSTNAME=db2whoc-flex-zipnqsp.services.au-syd.bluemix.net;PORT=50000;PROTOCOL=TCPIP;UID=bluadmin;PWD=zWG@U4q1uFpDTi0v8jVBDI7_PtSr0;";
 
  app.post('/newDataEntry', function(request, response){
-   console.log('NEW DATA API CALL:');
-   console.log(request);
+   console.log('NEW DATA API CALL:');   
```

Comment:
why still log this if not logging any details?

Reply:
I think its just a INFO log. We can remove it actually.